rusk/src/bin/config/chain.rs

```diff
@@ -9,6 +9,8 @@ use std::{path::PathBuf, time::Duration};
 use node::database::DatabaseOptions;
 use serde::{Deserialize, Serialize};
 
+pub const DEFAULT_BLOCK_GAS_LIMIT: u64 = 5 * 1_000_000_000;
```

Comment:
Can't we reduce definitions to a single one used everywhere?

Reply:
>I'm not familiar with this, but why do we use different values of this limit in tests and benches.
There's only one case where this is explained.

Not my decision, I cannot answer this question.